pytorch_blade/bazel/torch/torch.BUILD.tpl

```diff
@@ -0,0 +1,77 @@
+package(default_visibility = ["//visibility:public"])
+
+load(
+    "@local_config_cuda//cuda:build_defs.bzl",
+    "if_cuda_is_configured",
+)
+
+cc_library(
+    name = "libtorch",
```

Comment:
I'm wondering if it's a proper usage of cc_library. 
Can we use copts with -L/-I instead of adding a cc_library "libtorch" ?

Reply:
I think making the library a build target is better than using a link option. Because we can describe the dependencies in the target.